common/src/Polyhedron_Subtract.h

```diff
@@ -47,9 +47,10 @@ class Polyhedron<T,FP,VP>::Subtract {
     m_minuend(minuend),
     m_subtrahend(subtrahend),
     m_callback(callback) {
-        if (clipSubtrahend()) {
+        // FIXME: this early-out test is broken
+        //if (clipSubtrahend()) {
```

Comment:
How is it broken? Could you add a few words of explanation?

Reply:
Sorry, that was a useless comment.

I fixed the problem properly - it wasn't that `clipSubtrahend()` was broken, but we weren't saving the subtraction result to `m_fragments` in the case when `clipSubtrahend()` was returning false, meaning the minuend/subtrahend are disjoint.